sensu-plugins-xmpp.gemspec

```diff
@@ -36,7 +36,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'pry',                       '~> 0.10'
   s.add_development_dependency 'rake',                      '~> 10.5'
   s.add_development_dependency 'redcarpet',                 '~> 3.2'
-  s.add_development_dependency 'rubocop',                   '~> 0.40.0'
   s.add_development_dependency 'rspec',                     '~> 3.4'
+  s.add_development_dependency 'rubocop',                   '~> 0.52.1'
```

Comment:
At the time the CVE was reviewed 0.51 was the most recent version, we try to keep the versions consistent across the repos as much as possible as it makes it easier to know the cops better if you know which version of ruleset you are developing against. I won't force the change but I would prefer if we locked it to `~> 0.51.0` which locks it at the minor of 0.51 but allows new patches if they are released.